page-objects/administration.js

```diff
@@ -145,7 +146,7 @@ submitConsent: async function(e_mail) {
         })
 
         const randomStudent = allStudents.data[Math.floor(Math.random() * allStudents.data.length)]
-        const singleStudent = await Api.getStudent(jwt, randomStudent._id)
+        const singleStudent = await Api.getStudentAsAdmin(jwt, randomStudent._id)
```

Comment:
is it own student? probably better: const ownStudent?

Reply:
it try to get the student via admin route